app/forms/webauthn_setup_form.rb

```diff
@@ -110,6 +110,7 @@ def valid_attestation_response?(protocol)
     )
 
     begin
+      @aaguid = attestation_response.authenticator_data.aaguid
```

Comment:
Couple minor comments:

- If we're not expecting this to raise an exception, I think it'd be best to limit the `rescue`'d `begin` block to code which is expected to raise.
- Assigning it as a side-effect of validation feels like it creates a fragile relationship between assignment and use. Since we have `attestation_response` as a consumed parameter in `submit` and the value is derived from that, we could create a method which derives directly from that and avoids another instance variable.

```rb
def aaguid
  attestation_response&.authenticator_data&.aaguid
end
```